src/manifolds/Sphere.jl

```diff
@@ -1,81 +1,146 @@
+"""
+    AbstractSphere{𝔽} <: AbstractEmbeddedManifold{DefaultIsometricEmbeddingType}
+
+An abstract type to represent a unit sphere that is represented isometrically in the embedding.
+"""
+abstract type AbstractSphere{𝔽} <:
+    AbstractEmbeddedManifold{DefaultIsometricEmbeddingType} where {𝔽<:AbstractNumbers} end
+
+@doc raw"""
+    Sphere{n,𝔽} <: AbstractSphere{𝔽}
+
+The (unit) sphere manifold $𝕊^{n}$ is the set of all unit norm vectors in $𝔽^{N}$, $𝔽 elements. The sphere is
+represented in the embedding, and currently supports both vectors and matrices, i.e.
```

Comment:
I think representing points on a `Sphere` using matrices isn't really advisable (although it should more or less work in practice).

Reply:
Oh, that's a leftover from previous docs, I will omit the matrices part.